packages/axios-extension/src/abap/lrep-service.ts

```diff
@@ -98,6 +89,16 @@ function getNamespaceAsString(namespace: Namespace): string {
     return typeof namespace !== 'string' ? `apps/${namespace['reference']}/appVariants/${namespace['id']}/` : namespace;
 }
 
+/**
+ * Check if a variable is a buffer.
+ *
+ * @param input variable to be checked
+ * @returns true if the input is a buffer
+ */
+function isBuffer(input: string | Buffer): input is Buffer {
```

Comment:
Return type is `input is Buffer`? Is this some new notation? Why not use `boolean`?

Reply:
Return type is still boolean, that is a standard type guard (https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards)

Reply:
Thanks!